Library/Homebrew/formula_installer.rb

```diff
@@ -1208,7 +1232,7 @@ def fetch
     oh1 "Fetching #{Formatter.identifier(formula.full_name)}".strip
 
     if pour_bottle?(output_warning: true)
-      formula.fetch_bottle_tab
+      fetch_bottle_tab
```

Comment:
Isn't this redundant?

```
#fetch >> #fetch_dependencies >> #compute_dependencies >> #fetch_bottle_tab
```

Reply:
Technically yes but it's cached so calling twice has no negative effect and I felt like trying to maintain the existing flow was more resilient given how many possible branches there are/were here.

Reply:
Okay, as long as it was intentional.